models/org.go

```diff
@@ -749,23 +749,36 @@ func (org *User) AccessibleReposEnv(userID int64) (AccessibleReposEnvironment, e
 }
 
 func (org *User) accessibleReposEnv(e Engine, userID int64) (AccessibleReposEnvironment, error) {
+	var user *User
+
+	if userID > 0 {
+		u, err := getUserByID(e, userID)
+		if err != nil {
+			return nil, err
+		}
+		user = u
+	}
+
 	teamIDs, err := org.getUserTeamIDs(e, userID)
 	if err != nil {
 		return nil, err
 	}
 	return &accessibleReposEnv{
 		org:     org,
-		userID:  userID,
+		user:    user,
 		teamIDs: teamIDs,
 		e:       e,
 		orderBy: SearchOrderByRecentUpdated,
 	}, nil
 }
 
 func (env *accessibleReposEnv) cond() builder.Cond {
-	var cond builder.Cond = builder.Eq{
-		"`repository`.owner_id":   env.org.ID,
-		"`repository`.is_private": false,
+	var cond = builder.NewCond()
+	if env.user != nil && !env.user.IsRestricted {
```

Comment:
Shouldn't this be something like this?
```suggestion
	if env.user == nil || !env.user.IsRestricted {
```

Reply:
Obviously =)

Reply:
(fixed in d775c3b 👇)